mifosng-android/src/main/java/com/mifos/mifosxdroid/online/groupdetails/GroupDetailsFragment.kt

```diff
@@ -100,8 +107,29 @@ class GroupDetailsFragment : MifosBaseFragment() {
         binding.btnActivateGroup.setOnClickListener {
             onClickActivateGroup()
         }
+        binding.layoutError.findViewById<Button>(R.id.btnTryAgain).setOnClickListener {
+            reloadOnError()
+        }
+    }
+
+    fun reloadOnError() {
+        sweetUIErrorHandler?.hideSweetErrorLayoutUI(binding.scrollGroupDetails, binding.layoutError)
+        viewModel.loadGroupDetailsAndAccounts(groupId)
+
     }
 
+    private fun showErrorLayout() {
+        val errorMessage = getStringMessage(R.string.failed_to_load_group_details)
+        sweetUIErrorHandler?.showSweetErrorUI(
+            errorMessage, R.drawable.ic_error_black_24dp,
+            binding.scrollGroupDetails, binding.layoutError
+        )
+    }
+
+    private fun showUserInterface() {
+
+        sweetUIErrorHandler = SweetUIErrorHandler(requireActivity(), binding.root)
```

Comment:
Remove this method and initialize the sweetUIErrorHandler globally in onCreateView